app/src/main/java/com/activity_sync/renderers/base/AdapteeCollection.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2014 Pedro Vicente Gómez Sánchez.
```

Comment:
Is it part of come library ? Looks like copy-pasted from the web.
If this is a part of some external lib/entity can't it be used without copy-pasting ?